sdks/java/io/hadoop-file-system/src/test/java/org/apache/beam/sdk/io/hdfs/HadoopFileSystemTest.java

```diff
@@ -267,9 +289,15 @@ private void create(String relativePath, byte[] contents) throws Exception {
     }
   }
 
-  private byte[] read(String relativePath) throws Exception {
+  private byte[] read(String relativePath, long bytesToSkip) throws Exception {
     try (ReadableByteChannel channel = fileSystem.open(testPath(relativePath))) {
-      return ByteStreams.toByteArray(Channels.newInputStream(channel));
+      InputStream inputStream = Channels.newInputStream(channel);
+      if (bytesToSkip > 0) {
+        long bytesSkip = inputStream.skip(bytesToSkip);
```

Comment:
Use https://google.github.io/guava/releases/20.0/api/docs/com/google/common/io/ByteStreams.html#skipFully-java.io.InputStream-long-

Reply:
yes, did not not that one, better than manually skip + assert. Thanks Luke.